app/webpack/angularjs/components/medium/medium.test.js

```diff
@@ -12,12 +11,20 @@ describe("mau.directives.medium", function () {
   beforeEach(angular.mock.module("ngSanitize"));
 
   describe("with name and id", function () {
+    let scope;
+    let compile;
     let element;
+    let medium = { name: "my medium", id: 12, slug: "my-tag" };
 
     beforeEach(function () {
-      element = compileTemplate('<medium medium="medium"/>', {
-        medium: { name: "my medium", id: 12, slug: "my-tag" },
+      window.inject(function ($compile, $rootScope) {
```

Comment:
I wonder if this trick is better in general?  Should it be a helper?

I'm surprised (a bit) that we're calling `window.inject` on window and not `angular.mock.inject` which is what's in the helper method.

If this scheme is better or the current helper could be modified to support this (because that helper looks pretty close) maybe we have either 2 helpers (we'd need to name them to clearly understand
how they're different) or just make the current helper do the thing you need.

I might be inclined to try replacing the helper we have with what you have and see if other tests fail or pass.

With the move to jest, we also changed the dom in which angular is running.  So it might be that the old helper is just no good anymore.  Jest renders in `js-dom` and the previous stuff was using jasmine so it ran in either a real browser or phantom.js.  I could see that change warranting a change on the `compileTemplate` function we currently have.

Reply:
Yesterday I tried to use `angular.mock.inject` first but my IDE(RubyMine) weirdly alerted me that it was not defined and tests failed. So I just used `window.inject` directly. But today I tried again, it worked as well. I think I'll change it to `angular.mock.inject`.

Reply:
It might be that because we never explicitly import `angular.mock` but instead it's magically added on when you `import "angular-mocks"`, the IDE doesn't know about it.  not sure how that works, but glad the change worked.